CONTRIBUTING.rst

```diff
@@ -26,15 +26,16 @@ Pull requests are welcome, especially if they address open issues.
 We use flake8 to check some PEP8 errors/warnings. This will be checked with
 ``tox`` and Travis.
 
+Simple Bandwidth Scanner is moving towards 100% test coverage. Please help us
```

Comment:
We're at 68% currently, and i think we were already when you wrote that.
I'd rather say that any new feature should come with the test